djangocms_versioning/templates/admin/djangocms_versioning/change_list.html

```diff
@@ -2,12 +2,5 @@
 {% load i18n admin_urls %}
```

Comment:
You can get away probably by not loading `i18n` here as not needed in this template.